backend/tests/test_tasks_api.py

```
@@ -164,6 +167,17 @@ def source_message_id(task: TicketTask) -> str | None:
                     return None
                 return source_email.message_id
 
+            # To handle IN clause params safely in the mock
+            if related_email_id is None and "IN" in str(stmt).upper():
+                related_email_id = []
+                for k, v in params.items():
```

Ruff (0.15.14)

[warning] Loop control variable `k` not used within loop body

Rename unused `k` to `_k`

(B007)